README.md

```diff
@@ -4,25 +4,34 @@
 
 [<img src="https://github.com/MaterializeInc/materialize/assets/23521087/39270ecb-7ac4-4829-b98b-c5b5699a16b8" width=35%>](https://materialize.com)
 
-Materialize is a **data warehouse** purpose-built for **operational workloads**
-where an analytical data warehouse would be too slow, and a stream processor
-would be too complicated.
+Materialize is a cloud-native **data warehouse** purpose-built for **operational
+workloads** where an analytical data warehouse would be too slow, and a stream
+processor would be too complicated.
 
 Using SQL and common tools in the wider data ecosystem, Materialize allows you
 to build real-time automation, engaging customer experiences, and interactive
 data products that drive value for your business while reducing the cost of
 [data freshness](https://materialize.com/blog/decouple-cost-and-freshness/).
 
-## Sign up
+## Get started
 
 Ready to try out Materialize? [Sign up](https://materialize.com/register/) to get started! 🚀
 
+Have questions? We'd love to hear from you:
+  * [Join our Slack](https://materialize.com/s/chat)
+  * [Send us mail](https://materialize.com/contact/)
+
 ## About
 
 Materialize is designed to help you interactively explore your streaming data, perform analytics against live relational data, or increase data freshness while reducing the load of your dashboard and monitoring tasks. The moment you need a refreshed answer, you can get it in milliseconds.
 
 It focuses on providing correct and [consistent](https://materialize.com/docs/overview/isolation-level/) answers with minimal latency, and does not ask you to accept either approximate answers or eventual consistency. Whenever Materialize answers a query, that answer is the correct result on some specific (and recent) version of your data. Materialize does all of this by recasting your SQL queries as *dataflows*, which can react efficiently to changes in your data as they happen.
 
+Our fully managed service is cloud native, featuring **high availability**, via
+multi-active replication, **horizontal scalability**, by seamlessly scaling
+dataflows across multiple machines, and **near infinite storage**, by
```

Comment:
```suggestion
compute across multiple machines, and **near infinite storage**, by
```

Reply:
Oh, I didn't re-read the whole thing. We could remove references to dataflows and more generically use "compute", which doesn't presuppose knowledge of Materialize internals; but not something I feel strongly about.

Reply:
Yeah, might be worth doing, but also I don't mind that the GitHub README goes a little deeper into the technical weeds than other things. Just gonna leave as is for now and let someone else address in the future.